### PR TITLE
ggplot2 3.5.0 axes blog

### PR DESCRIPTION
**Pull request by teunbrand**
This blog accompanies the main release blogpost and is a post about axes.
It was split off from #674.
Pinging @thomasp85 for review.

### CODE REVIEW

File: content/blog/ggplot2-3-5-0-axes/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+* [x] `hugodown::use_tidy_thumbnails()`
+* [ ] Add intro sentence, e.g. the standard tagline for the package
+* [ ] `usethis::use_tidy_thanks()`
+-->
+
+We are pleased to release [ggplot2](https://ggplot2.tidyverse.org) 3.5.0.
+This is one blogpost among several outlining changes to axes.
+Please find the [main release post](/blog/2024/02/ggplot2-3-5-0/) to read about other changes.
+
+Axes, alongside [legends](/blog/2024/02/ggplot2-3-5-0-legends/), are visual representations of scales and allow observes to translate graphical properties of a plot into information.
```
Comment (hadley) [inline content/blog/ggplot2-3-5-0-axes/index.Rmd:38]:
```suggestion
Axes, alongside [legends](/blog/2024/02/ggplot2-3-5-0-legends/), are visual representations of scales and allow observers to read values off of a plot.
```

File: content/blog/ggplot2-3-5-0-axes/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+ggplot(mpg, aes(displ, hwy)) +
+  geom_point() +
+  guides(
+    x = guide_axis(cap = "both"), # Cap both ends
+    y = guide_axis(cap = "upper") # Cap the upper end
+  ) +
+  theme(axis.line = element_line())
+```
+
+## Logarithmic axes
```
Comment (hadley) [inline content/blog/ggplot2-3-5-0-axes/index.Rmd:112]: These look great!

File: content/blog/ggplot2-3-5-0-axes/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+The first axis is placed next to the panel and subsequent axes are placed further away from the panel.
+Axes, like legends, have acquired a `theme` argument that can be used to customise the display of individual axes.
+
+```{r stack_axis}
+#| fig.alt: >
+#|  Scatterplot of engine displacement versus highway miles per 
+#|  gallon. The x-axis resembles railroad tracks with labels in the middle and
+#|  an additional line beneath.
+
+ggplot(mpg, aes(displ, hwy)) +
```
Comment (hadley) [inline content/blog/ggplot2-3-5-0-axes/index.Rmd:174]: Do you have a more compelling example to show off here? I don't get why people would want this.

**Review by hadley — APPROVED** [review]

File: content/blog/ggplot2-3-5-0-axes/index.Rmd (hunk 3 above)
Comment (thomasp85) [reply content/blog/ggplot2-3-5-0-axes/index.Rmd:174]: I think this is more compelling once guide extensions arrives - i.e. you could stack a discrete axis with a table giving different traits for the various levels... Or you could have a continuous axis with qualitative axis below to explain what the limits of the scale means (bad <-> good)... But I don't know off-hand of a compelling example that can be made with just the standard axes available from ggplot2

Comment (teunbrand) [reply content/blog/ggplot2-3-5-0-axes/index.Rmd:174]: I also don't think there is a compelling example to make with just the axes in ggplot2.

To give an example with some WIP guide extension, you could imagine a guide that draws labelled brackets at some interval, which would be a separate guide, but can be combined with the standard axis.

image

Comment (hadley) [reply content/blog/ggplot2-3-5-0-axes/index.Rmd:174]: In that case, I think you might just mention this without an example, highlighting that it's an important building block for future work.

**Pull request merged**